modules/release-notes/pages/relnotes.adoc

```diff
@@ -546,6 +546,8 @@ WARNING: Changing the internal replication setting will cause all XDCR processes
 | https://issues.couchbase.com/browse/MB-32685[MB-32685^]
 | *Summary*: Introduced a delay in updating the RBAC database revision number to avoid returning AUTH_STALE during a refresh of the RBAC database.
 
+| https://issues.couchbase.com/browse/MB-32497[MB-32497^]
+| *Summary*: Connection::signalIfIdle may cause a deadlock when used from a different thread
```

Comment:
Think we need to add more explanation to what the user might see e.g.
"Memcached connections may hang due to Connection::signalIfIdle triggering a deadlock when used from a different thread" Think adding the words connection and hang will help to make it easier for people when searching though the release notes :)